packages/desktop-client/src/components/accounts/AccountSyncCheck.js

```diff
@@ -145,10 +143,12 @@ function AccountSyncCheck({
   );
 }
 
-export default connect(
-  state => ({
-    accounts: state.queries.accounts,
-    failedAccounts: state.account.failedAccounts,
-  }),
-  actions,
-)(AccountSyncCheck);
+export default withRouter(
```

Comment:
Could this use the hook API instead?

Reply:
Sure: https://github.com/actualbudget/actual/pull/1133/commits/8dfb62db709be1dca53e813da1c4df2704e5ead7